### DISCUSSION TITLE
herokuku回档问题

### DISCUSSION THREAD

**Discussion by sw586** [libsgh/PanIndex#181] [General]
使用herokuku昨天明明修改了密码，添加了4个网盘，第二天看只有2个网盘，密码也回档为默认密码了，不知道是什么问题

**Comment by libsgh** [comment]
heroku本身是会重置的，所以数据和配置会丢失，配置可以使用环境变量的形式避免丢失，缓存目录丢失没有什么好办法，如果文件数量较少 可以设置下定时任务，每天执行几次缓存任务。最后还是建议使用vps部署。
